test/fixtures/transfers_collection.json

```diff
@@ -0,0 +1,40 @@
+{
+    "entity": "collection",
+    "count": 1,
+    "items": [
+      {
+        "id": "trf_J0FrZYPql4riDx",
+        "entity": "transfer",
+        "status": "processed",
+        "source": "pay_I6i1edcUUytu9p",
+        "recipient": "acc_I0QRP7PpvaHhpB",
+        "amount": 100,
+        "currency": "INR",
+        "amount_reversed": 0,
+        "fees": 1,
+        "tax": 0,
+        "notes": [
+  
+        ],
+        "linked_account_notes": [
+  
+        ],
```

Comment:
Pls make the samples consistent with docs.

Reply:
Done